### DISCUSSION TITLE
Reading aedats event-wise

### DISCUSSION THREAD

**Discussion by gokulbnr** [neuromorphs/tonic#287] [General]
Is it possible to stream events one-by-one from the aedat files without loading the whole data onto the RAM all at once before processing it? I have large aedats with DVS, APS, and IMU events recorded which I am trying to decode with a modified read_davis_346() function but the memory is becoming a major constraint here.

**Comment by biphasic** [comment]
For that I would recommend the https://github.com/neuromorphicsystems/aedat package!
